common/core/nesteddisplay.cpp

```diff
@@ -207,4 +410,48 @@ bool NestedDisplay::GetDisplayName(uint32_t *size, char *name) {
   return true;
 }
 
+#ifdef NESTED_DISPLAY_SUPPORT
```

Comment:
We can remove this check ?

Reply:
I suggest keep this check, while in native android we don't need these data struct and the thread instance.